From 61ef785ed96a227d43b1b0696fdfbccb3d5db5ea Mon Sep 17 00:00:00 2001
From: Tormod Volden <debian.tormod@gmail.com>
Date: Fri, 6 May 2016 10:47:41 +0200
Subject: [PATCH] Correct top-level directory name in repackaged tarballs

Signed-off-by: Tormod Volden <debian.tormod@gmail.com>
---

I believe the top-level directory name in a repackaged source tarball
should be name-ver+dfsg and not name-ver.orig

Best regards,
Tormod

 best-pkging-practices.dbk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/best-pkging-practices.dbk b/best-pkging-practices.dbk
index 1651ddb..bbcf9f9 100644
--- a/best-pkging-practices.dbk
+++ b/best-pkging-practices.dbk
@@ -1746,7 +1746,7 @@ mirror rather than trying to locate a canonical upstream distribution point).
 <listitem>
 <para>
 <emphasis role="strong">should</emphasis> use
-<filename><replaceable>packagename</replaceable>-<replaceable>upstream-version</replaceable>.orig</filename> as the
+<filename><replaceable>packagename</replaceable>-<replaceable>upstream-version</replaceable>+dfsg</filename> as the
 name of the top-level directory in its tarball.  This makes it possible to
 distinguish pristine tarballs from repackaged ones.
 </para>
-- 
1.9.1

